client/js/components/map/admin/AdminLayerListItem.vue

```diff
@@ -108,7 +108,7 @@
         <input
           type="checkbox"
           data-cy="adminLayerListItem:toggleDefaultVisibility"
-          :disabled="'' !== layer.attributes.visibilityGroupId || (true === isChildOfCategoryThatAppearsAsLayer)"
+          :disabled="(!layer.attributes.visibilityGroupId || ('' !== layer.attributes.visibilityGroupId && null !== layer.attributes.visibilityGroupId)) || (true === isChildOfCategoryThatAppearsAsLayer)"
```

Comment:
Wouldn't this be enough? `!layer.attributes.visibilityGroupId` checks for both empty string and `null`

```suggestion
          :disabled="(!layer.attributes.visibilityGroupId || (true === isChildOfCategoryThatAppearsAsLayer)"
```